lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveAdmin
+  class ErrorHandler
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      @app.call(env)
+    rescue Exception => e
+      if ActiveAdmin.application.default_namespace.to_s == env['PATH_INFO'].split('/').second
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.